backend/functions/lib/adapters/gateways/case.assignment.cosmosdb.repository.ts

```diff
@@ -51,22 +52,54 @@ export class CaseAssignmentCosmosDbRepository implements CaseAssignmentRepositor
     }
   }
 
+  async updateAssignment(caseAssignment: CaseAttorneyAssignment): Promise<string> {
+    try {
+      const { item } = await this.cosmosDbClient
+        .database(this.cosmosConfig.databaseName)
+        .container(this.containerName)
+        .item(caseAssignment.id)
+        .replace(caseAssignment);
+      log.debug(this.applicationContext, MODULE_NAME, `Assignment updated ${item.id}`);
+      return item.id;
+    } catch (e) {
+      log.error(this.applicationContext, MODULE_NAME, `${e.status} : ${e.name} : ${e.message}`);
+      if (e.status === 403) {
+        throw new ForbiddenError(MODULE_NAME, {
+          message:
+            'Unable to update assignment. Please try again later. If the problem persists, please contact USTP support.',
+          originalError: e,
+          status: 500,
+        });
+      } else {
+        throw new UnknownError(MODULE_NAME, {
+          message:
+            'Unable to update assignment. Please try again later. If the problem persists, please contact USTP support.',
+          originalError: e,
+          status: 500,
+        });
+      }
+    }
+  }
+
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   getAssignment(assignmentId: string): Promise<CaseAttorneyAssignment> {
     throw new Error('Method not implemented.');
   }
 
   async findAssignmentsByCaseId(caseId: string): Promise<CaseAttorneyAssignment[]> {
+    let query = '';
+    query = 'SELECT * FROM c WHERE c.caseId = @caseId AND NOT IS_DEFINED(c.unassignedOn)';
```

Comment:
```suggestion
    const query = 'SELECT * FROM c WHERE c.caseId = @caseId AND NOT IS_DEFINED(c.unassignedOn)';
```
`let` was used before because there was some conditional statements to update the query. We could refactor this way since conditional been remoted.